modules/get-started/pages/whats-new-233.adoc

```diff
@@ -0,0 +1,26 @@
+= What's New in 23.3 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 23.3 Beta. 
+
+== OIDC authentication
+
+xref:manage:security/console/generic-oidc.adoc[OIDC authentication] provides single sign-on capabilities in Redpanda and Redpanda Console to use external accounts instead of SASL users. With this, you can manage user and client/application credentials in the identity provider, such as Okta, Azure AD, and on-premise Active Directory Federation Service (AD FS).
+
+== Audit logging 
+
+Redpanda's xref:manage:audit-logging.adoc[audit logging] supports fine-grained auditing of Kafka API and ACL operations. You can persistently store auditing events to a topic in an append-only fashion, which guarantees retention for the requested time period, even against Redpanda administrators. Auditing is compatible with the Open Cybersecurity Schema Framework (OCSF) to provide a predictable and extensible solution with industry standard tools, such as Splunk, Sumo Logic, and AWS.
```

Comment:
@deniscoady can you reword what operations are audited?

Reply:
I've created a list of high-level auditing capabilities below. Have I missed any major ones @michael-redpanda?

------
Redpanda's audit logging capability supports fine-grained reporting of events in the Kafka, Pandaproxy, Schema Registry, and Admin APIs. Some of these events include:
 - changes to users or user ACLs
 - changes to cluster, users, topics, ACLs, or consumer groups.
 - describing a cluster, user, topic, ACL, or consumer group.
 - whenever a user is authenticated 
 - whenever a topic is being written to or read from (requires explicit opt-in)
 - HTTP requests for Schema Registry, Pandaproxy, and Admin API.